pages/common/clamav.md

```diff
@@ -0,0 +1,17 @@
+# ClamAV
+
+> Open-source anti-virus program.
+> ClamAV isn't a command, but a set of commands. See the pages below.
```

Comment:
```suggestion
> ClamAV isn't a command, but a set of commands.
```
Should we mention `See the pages below separately`, I think it would be self-explanatory.